sublime-package.json

```diff
@@ -0,0 +1,241 @@
+                      "default": true,
+                      "description": "Flags long, complicated sentences that could potentially confuse your reader.",
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.ReadabilityTransforms": {
+                      "default": true,
+                      "description": "Suggests splitting long, complicated sentences that could potentially confuse your reader.",
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.SentenceVariety": {
+                      "default": true,
+                      "description": "Flags series of sentences that follow the same pattern.",
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.SpacesSurroundingSlash": {
+                      "default": true,
+                      "description": "Suggests removing extra spaces surrounding a slash.",
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.SplitInfinitive": {
+                      "default": true,
+                      "description": "Suggests rewriting split infinitives so that an adverb doesn't come between 'to' and the verb.",
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.StylisticFragments": {
+                      "default": false,
+                      "description": "Suggests completing all incomplete sentences, including stylistic sentence fragments that may be intentional.",
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.UnnecessaryEllipses": {
+                      "default": false,
+                      "description": "Flags unnecessary use of ellipses (...).",
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.Variety": {
+                      "default": true,
+                      "description": "Suggests alternatives to words that occur frequently in the same paragraph.",
+                      "type": "boolean"
+                    },
+                    "grammarly.config.suggestions.Vocabulary": {
+                      "default": true,
+                      "description": "Suggests alternatives to bland and overused words such as 'good' and 'nice'.",
+                      "type": "boolean"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      {
+        "file_patterns": [
+          "/*.sublime-project"
+        ],
+        "schema": {
+          "properties": {
+            "settings": {
+              "properties": {
+                "LSP": {
+                  "properties": {
+                    "LSP-Grammarly": {
```

Comment:
Because server name is overridden to `grammarly`, this actually needs to be the same.